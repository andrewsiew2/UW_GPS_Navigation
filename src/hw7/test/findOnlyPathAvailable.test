# Find a path with Dijkstra's algorithm when there is only one possible path
# Loads data from testing.tsv

LoadGraph test testing.tsv

FindPath test DOOM INVOKER
