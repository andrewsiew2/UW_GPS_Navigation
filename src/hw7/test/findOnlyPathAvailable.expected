# Find a path with Dijkstra's algorithm when there is only one possible path
# Loads data from testing.tsv

loaded graph test

path from DOOM to INVOKER:
DOOM to NECRO with weight 0.500
NECRO to INVOKER with weight 1.000
total cost: 1.500